src/pages/dataElements/form/DataElementFormFields.tsx

```diff
@@ -230,6 +219,6 @@ export function DataElementFormFields() {
                     />
                 </StandardFormSection>
             )}
-        </>
+        </Loader>
     )
```

Comment:
Can't comment on the exact line since it wasn't changed in this PR. 

```
customAttributes.data?.length && (
```

This will render 0 if there are no attributes. Please cast to a Boolean, compare the actual length size or use ternary.